infrastructure/code/convert.go

```diff
@@ -391,12 +392,46 @@ func (s *SarifConverter) toIssues(baseDir string) (issues []snyk.Issue, err erro
 				CWEs:                testRule.Properties.Cwe,
 			}
 
+			d.IsIgnored, d.IgnoreDetails = s.getIgnoreDetails(result)
 			issues = append(issues, d)
 		}
 	}
 	return issues, errs
 }
 
+func (s *SarifConverter) getIgnoreDetails(result codeClient.Result) (bool, *snyk.IgnoreDetails) {
+	isIgnored := false
+	var ignoreDetails *snyk.IgnoreDetails
+
+	// this can be an array of multiple suppressions in SARIF
+	// but we only store one ignore for now
+	if len(result.Suppressions) == 1 {
```

Comment:
Hm... for resiliency, shouldn't this be `> 0` and then only take the first element for ignoring, maybe logging that there are more?

Reply:
That's a good idea!